.github/dependabot.yml

```diff
@@ -3,6 +3,7 @@ enable-beta-ecosystems: true
 updates:
   - package-ecosystem: "github-actions"
     directory: "/"
+    commit-message: "GitHub Actions update"
```

Comment:
I'm not sure if this is syntactically correctly but I can't find any dependabot validator